skrypty/utils/config/conf_loader.lua

```diff
@@ -47,6 +47,7 @@ end
 function scripts_load_config(name)
     scripts.config_loader.last_config = name
     dofile(scripts.config_loader:get_config_path(name))
+    raiseEvent("profileLoaded")
```

Comment:
warto zrzucac `imie` jako parametr?

Reply:
Jak już ktoś bardzo będzie potrzebował to jest `scripts.config_loader.last_config`